src/app/services/project.service.ts

```diff
@@ -0,0 +1,78 @@
+import { inject, injectable } from 'tsyringe';
+import { IProjectRepository } from '../../domain/IRepos/IProjectRepo';
+import { CreateProjectDto, UpdateProjectDto, ProjectResponseDto } from '../../domain/DTOs/projectDTO';
+
+@injectable()
+export class ProjectService {
+    constructor(
+        @inject('IProjectRepo') private readonly ProjectRepo: IProjectRepository
```

Comment:
```suggestion
        @inject('IProjectRepo') private readonly projectRepo: IProjectRepository
```
follow JS naming convention